### PR TITLE
various fixes / cleanups / renames

### PR DESCRIPTION
**Pull request by mariakleiner**
post #232 and #233 merge

### CODE REVIEW

File: core/ts/recipe/Graphinator.ts [hunk 1]
```diff
@@ -170,7 +170,7 @@ export class Graphinator {
   findRemovedParticles(particles) {
     const runningGraphParticleIds = keys(this.arc.hosts).filter(id => {
       const container = this.arc.hosts[id].meta.container;
-      if (container === 'main#graph') {
+      if (container === 'designer#graph') {
```
Comment (sjmiles) [inline core/ts/recipe/Graphinator.ts:173]: note: there is no 'universal root' right now, so this is a corner we are cutting (also other places)

File: pkg/Apps/Graph/Library/GraphBuilderNode.js [hunk 2]
```diff
@@ -13,9 +13,9 @@ const globalStores = [
   'nodeId',
 ];
 
-export const RecipeBuilderRecipe = {
+export const GraphBuilderNode = {
```
Comment (sjmiles) [inline pkg/Apps/Graph/Library/GraphBuilderNode.js:16]: Can't be a node unless it's has a $meta and an id. I think suspect these objects in Graph/Library simply still Recipes.

Actual Nodes for these things are in (main) Library/Node.

File: pkg/Apps/Graph/Library/GraphToolbarNode.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
  * Use of this source code is governed by a BSD-style
  * license that can be found in the LICENSE file.
  */
-export const GraphToolbarRecipe = {
```
Comment (sjmiles) [inline pkg/Apps/Graph/Library/GraphToolbarNode.js:7]: As above, but GraphToolbarNode is in NodeGraph/GraphToolbarNode.

File: pkg/Apps/Graph/Library/InspectorNode.js [hunk 4]
```diff
@@ -6,7 +6,7 @@
  */
 import {customInspectors} from './customInspectors.js';
 
-export const InspectorRecipe = {
+export const InspectorNode = {
```
Comment (sjmiles) [inline pkg/Apps/Graph/Library/InspectorNode.js:9]: Not a node?

File: pkg/Apps/Graph/Library/NodeEditorNode.js [hunk 5]
```diff
@@ -4,7 +4,7 @@
  * Use of this source code is governed by a BSD-style
  * license that can be found in the LICENSE file.
  */
-export const NodeEditorRecipe = {
+export const NodeEditorNode = {
```
Comment (sjmiles) [inline pkg/Apps/Graph/Library/NodeEditorNode.js:7]: Not a node?

File: pkg/Apps/Graph/Library/NodegraphGraph.js [hunk 6]
```diff
@@ -5,10 +5,10 @@
  * license that can be found in the LICENSE file.
  */
 import {NodeCatalogNode} from '../../../Library/Node/NodeCatalogNode.js';
```
Comment (sjmiles) [inline pkg/Apps/Graph/Library/NodegraphGraph.js:7]: I'm sorry I wasn't clear on the path forward.

I expect the entire Graph app is going to just go away in favor of GraphMaker, there is no reason to update these internal files.

File: pkg/Library/App/Nodejs/App.js [hunk 7]
```diff
@@ -8,7 +8,7 @@
  */
 import {Arcs} from './Arcs.js';
 //import {loadCss} from '../../Dom/dom.js';
-import {DevToolsRecipe} from '../../DevTools/DevToolsRecipe.js';
+import {DevToolsNode} from '../../DevTools/DevToolsNode.js';
```
Comment (sjmiles) [inline pkg/Library/App/Nodejs/App.js:11]: Don't make it a Node if it's used as a Recipe.

File: pkg/Library/App/Nodejs/App.js [hunk 8]
```diff
@@ -37,7 +37,7 @@ export const App = class {
     });
     //await loadCss(`${this.paths.$library ?? '.'}/Dom/Material/material-icon-font/icons.css`);
     // TODO(sjmiles): pick a syntax
-    const recipes = [DevToolsRecipe, ...(this.recipes ?? this.recipes ?? [])];
+    const recipes = [DevToolsNode, ...(this.recipes ?? this.recipes ?? [])];
```
Comment (sjmiles) [inline pkg/Library/App/Nodejs/App.js:40]: It says _recipes_.

File: pkg/Library/App/Worker/App.js [hunk 9]
```diff
@@ -43,7 +43,7 @@ export const App = class {
     this.composer = await Arcs.createComposer(this.root || document.body);
     //
     assign(this.nodeTypes, {
-      DevToolsRecipe,
+      DevToolsNode,
```
Comment (sjmiles) [inline pkg/Library/App/Worker/App.js:46]: This one is correctly a Node.

File: pkg/Library/DevTools/DevToolsGraph.js [hunk 10]
```diff
@@ -6,7 +6,7 @@
  * license that can be found in the LICENSE file or at
  * https://developers.google.com/open-source/licenses/bsd
  */
-export const DevToolsRecipe = {
+export const DevToolsNode = {
```
Comment (sjmiles) [inline pkg/Library/DevTools/DevToolsGraph.js:9]: Not a node.

File: pkg/Library/Media/DeviceUxNodes.js [hunk 11]
```diff
@@ -4,7 +4,7 @@
  * license that can be found in the LICENSE file.
  */
 
-export const DeviceUxRecipe = {
+export const DeviceUxNode = {
```
Comment (sjmiles) [inline pkg/Library/Media/DeviceUxNodes.js:7]: As above.

**Review by sjmiles — CHANGES_REQUESTED** [review]

File: core/ts/recipe/Graphinator.ts (hunk 1 above)
Comment (mariakleiner) [reply core/ts/recipe/Graphinator.ts:173]: yes :( acknowledged

File: pkg/Apps/Graph/Library/GraphBuilderNode.js (hunk 2 above)
Comment (mariakleiner) [reply pkg/Apps/Graph/Library/GraphBuilderNode.js]: i reverted all the recipe->node renaming changes... 
i'm not sure we should keep the word "recipe" in our glossary, there are probably "node types" now... let's leave it as is for now and discuss?

File: pkg/Apps/Graph/Library/NodegraphGraph.js (hunk 6 above)
Comment (mariakleiner) [reply pkg/Apps/Graph/Library/NodegraphGraph.js]: we should probably have a submitted `graph` file then, right?

File: pkg/Library/App/Nodejs/App.js (hunk 8 above)
Comment (mariakleiner) [reply pkg/Library/App/Nodejs/App.js]: i think this file is outdated, we should no longer have addRecipes API... or do we want to keep them both? i'm confused 😕

**Comment by mariakleiner** [comment]
renamed back... for some reason i keep fixing the DB app, should probably let if go and/or convert to a proper graph based app? anyway, can do it later.
PTAL

File: pkg/Apps/Graph/Library/GraphBuilderNode.js (hunk 2 above)
Comment (sjmiles) [reply pkg/Apps/Graph/Library/GraphBuilderNode.js]: Sgtm.

File: pkg/Library/App/Nodejs/App.js (hunk 8 above)
Comment (sjmiles) [reply pkg/Library/App/Nodejs/App.js]: You are right, this file is outdated. But IMO we should either fix it all the way, discard it, or leave it alone. Generally the notion of converting Recipes to Graphs is not wrong, it's just I think it's ok if we do it kind of slowly.

**Comment by sjmiles** [comment]
> for some reason i keep fixing the DB app, should probably let if go and/or convert to a proper graph based app? anyway, can do it later. PTAL

Yes! 

Sorry, I was not clear about this either. Generally anything using the 'old way' can be ignored until (1) we can triage them, (2) one of use wants to keep one alive / or upgrade it directly (aka Graph!).

At least this is my suggested path forward.

**Review by sjmiles — APPROVED** [review]

**Pull request merged**